mirrord/config/configuration.md

```diff
@@ -1106,16 +1106,63 @@ Similarly, you can exclude certain paths using a negative look-ahead:
 Setting this filter will make mirrord only steal requests to URIs that do not start with
 "/health/".
 
-#### feature.network.incoming.http_filter.all_of {#feature-network-incoming-http_filter-all_of}
+You can use multiple HTTP filters at the same time like this:
+```json
+{
+  "all_of": [
+    { "header": "^User-Agent: (?!kube-probe)" },
+    { "path": "^/api/" }
+  ]
+}
+```
+Filter above might be used to steal HTTP API requests, but not Kubernetes probes.
+
+You can also capture HTTP requests that match at least one of a set of filters like this:
+```json
+{
+  "any_of": [
+    { "path": "^/api/v1" }
+    { "path": "^/api/v2"}
+  ]
+}
+```
+Filter above might be used to steal only HTTP requests made to your application's API in
+versions 1 and 2.
```

Comment:
```suggestion
- A filter that requires at least **one** HTTP request to match, using `any_of`, this filter steals only HTTP requests made to your application's API in versions 1 or 2, excluding anything else:

```json
{
  "any_of": [
    { "path": "^/api/v1" }
    { "path": "^/api/v2"}
  ]
}
```

This one is a bit weird, since it's not really `1 or 2`, it's more a `1 or 2 excluding the rest`. Can you think of a better way to phrase this than my suggestion?

Reply:
Rephrased a bit, wdyt now?